### PR TITLE
Integers to be read as float

### PR DESCRIPTION
**Pull request by aensidhe**
Closes https://github.com/progaudi/tarantool-csharp/issues/92

### CODE REVIEW

File: src/msgpack.light/MsgPackContext.cs [hunk 1]
```diff
@@ -11,46 +11,49 @@ public class MsgPackContext
...
 
         private readonly Dictionary<Type, Type> _genericConverters = new Dictionary<Type, Type>();
 
         private readonly Dictionary<Type, Func<object>> _objectActivators = new Dictionary<Type, Func<object>>();
 
-        public MsgPackContext()
+        public MsgPackContext(bool strictParseOfFloat = false)
         {
+            var numberConverter = new NumberConverter(strictParseOfFloat);
+            _converters = new Dictionary<Type, IMsgPackConverter>
```
Comment (Dmitry-Bryliuk) [inline src/msgpack.light/MsgPackContext.cs:23]: make 2 static instances?

File: tests/msgpack.light.tests/msgpack.light.tests.csproj [hunk 2]
```diff
@@ -2,36 +2,29 @@
...
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    <PackageReference Include="xunit" Version="2.2.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
+    <PackageReference Include="JetBrains.Annotations" Version="10.3.0" />
     <PackageReference Include="Shouldly" Version="2.8.2" />
     <ProjectReference Include="..\..\src\msgpack.light\msgpack.light.csproj" />
   </ItemGroup>
-
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```
Comment (roman-kozachenko) [inline tests/msgpack.light.tests/msgpack.light.tests.csproj:28]: What kind of service is it?

**Review by roman-kozachenko — APPROVED** [review]

File: src/msgpack.light/MsgPackContext.cs (hunk 1 above)
Comment (roman-kozachenko) [reply src/msgpack.light/MsgPackContext.cs:23]: Instances can't be static.
Each context instance should have its own set of converters.

File: tests/msgpack.light.tests/msgpack.light.tests.csproj (hunk 2 above)
Comment (aensidhe) [reply tests/msgpack.light.tests/msgpack.light.tests.csproj:28]: @roman-kozachenko some kind of vs stuff.

**Pull request merged**